steps/27/README.md

```diff
@@ -85,75 +85,109 @@ We create a new `unitTests.qunit.ts` file under `webapp/test/unit/`.
 This script loads and executes our formatter.
 
 ```ts
-/* @sapUiRequire */
-QUnit.config.autostart = false;
-
-// import all your QUnit tests here
-void Promise.all([
-	import("ui5/walkthrough/test/unit/model/formatter")
-]).then(() => {
-	QUnit.start();
-});
+import "./model/formatter";
 ```
 
-> 📝 **Note:** <br>
-> The annotation `@sapUiRequire` instructs the UI5 TypeScript transpilation process (executed by `ui5-tooling-transpile`) to use `sap.ui.require` instead of `sap.ui.define` for the transpiled module. This allows to load the module via a `<script>` tag. This is important for test suites to guarantee that the `QUnit.config.autostart` is set to `false` directly after QUnit has been loaded to avoid that QUnit immediately starts the test execution before the QUnit tests have been imported. Once the QUnit tests have been imported the tests will be executed after calling `QUnit.start()`.
-
 ***
 
-### webapp/test/unit/unitTests.qunit.html \(New\)
-
-Finally we create a new `unitTests.qunit.html` page under `webapp/test/unit`.
-
-Since we are now in the `webapp/test/unit` folder, we actually need to go up two levels to get the `webapp` folder again. This namespace can be used inside the tests to load and trigger application functionality.
+### webapp/test/testsuite.qunit.html \(New\)
 
-First, we load some basic QUnit functionality via script tags. The QUnit test suite must be included at the end via a script tag which loads `unitTests.qunit.js`. The file extension `.js` must be used since this loads the transpiled version of `unitTests.qunit.ts`.
+We create a new `testsuite.qunit.html` page under `webapp/test`. 
+This page serves as an entry point for all tests within our project. For more information about the UI5 Test Suite, read [Concept and Basic Setup](https://sdk.openui5.org/#/topic/22f50c0f0b104bf3ba84620880793d3f).
 
```

Comment:
At least for me as a beginner, this is not enough to understand the implementation below: Why script id)"sap-ui-bootstrap" is needed here? What is "../resources/sap/ui/test/starter/createSuite.js" for? Why do I need a new namespace "test-resources.ui5.walkthrough"? Does it need to be test-resources+namespace of my initial app or could this be any name?
What is the configuration option "data-sap-ui-testsuite" for and how to define it?
Was does these two divs at the end to? To which id="qunit" and id="qunit-fixture" are referencing?
...?

It seems as it would make sene to create this file only after testsuite.qunit.ts was created, so learners understand on which this value is based.